ci-operator/step-registry/ipi/conf/ibmcloud/custom-endpoints/ipi-conf-ibmcloud-custom-endpoints-commands.sh

```diff
@@ -59,3 +124,12 @@ fi
 if [ -n "$SERVICE_ENDPOINT_GlobalTagging" ]; then
   patch_endpoint "GlobalTagging" $SERVICE_ENDPOINT_GlobalTagging
 fi
+if [[ -n "$SERVICE_ENDPOINT_COSConfig" && ${isPre418} -eq 1 ]]; then
```

Comment:
when `SERVICE_ENDPOINT_COSConfig` is specified by users, that means users want to overwrite the endpoint on purpose, it does not depends on OCP version.